src/tbs_matchmaking_server.cpp

```diff
@@ -33,10 +33,8 @@
 #include <iostream>
 
 #include <sys/types.h>
-#ifdef __linux__
```

Comment:
I didn't knew Microsoft is POSIX at last :smiley:.

Reply:
_bentley wrote_:
> POSIX specifies that waitpid() requires sys/wait.h, so include it everywhere.

Reply:
If it causes problems on Windows, “ifndef windows” might be more appropriate than rolling it back.